server/utils/azureAppInsights.ts

```diff
@@ -54,3 +62,19 @@ export function appInsightsMiddleware(): RequestHandler {
     next()
   }
 }
+
+export function addUserDataToRequests(envelope: EnvelopeTelemetry, contextObjects: ContextObject | undefined): boolean {
```

Comment:
Does this also add the activeCaseloadId? I think the ticket asks for that as well 🤔